components/cli/pkg/util/utils.go

```diff
@@ -514,3 +522,86 @@ func GetFileSize(path string) (int64, error) {
 
 	return file.Size(), nil
 }
+
+func RenameFile(oldName, newName string) error {
```

Comment:
The function name should start with a simple letter.

Reply:
Discussed.